<launch>
  <node pkg="nodelet" type="nodelet" name="nodelet_manager"  args="manager" output="screen"/>
 
  <node pkg="nodelet" type="nodelet" name="drv_nodelet" args="load fisheye_drv/drv_nodelet nodelet_manager" output="screen">
    <param name="back_number" value="0"/>
    <param name="front_number" value="1"/>
    <param name="left_number" value="2"/>
    <param name="right_number" value="3"/>
        
    <param name="back_topic" value="/driver/fisheye/back"/>
    <param name="front_topic" value="/driver/fisheye/front"/>
    <param name="left_topic" value="/driver/fisheye/left"/>
    <param name="right_topic" value="/driver/fisheye/right"/>
  </node>
</launch>